packages/ui-gui/package.json

```diff
@@ -1,28 +1,38 @@
 {
   "name": "@webpack-cli/ui-gui",
   "version": "0.0.1",
+  "license": "MIT",
   "description": "GUI for webpack-cli ui",
-  "main": "index.js",
+  "main": "src/index.js",
   "scripts": {
-    "build": "webpack --config=webpack.prod.js",
-    "serve": "webpack-dev-server --config=webpack.prod.js"
+    "start": "if-env NODE_ENV=production && npm run -s serve || npm run -s dev",
+    "build": "preact build",
+    "serve": "preact build && preact serve",
+    "dev": "preact watch",
+    "lint": "eslint src"
   },
-  "license": "MIT",
+  "eslintConfig": {
+    "extends": "eslint-config-synacor",
+    "rules": {
+      "no-unused-vars": "warn",
+      "react/sort-comp": "off",
+      "lines-around-comment": "off",
+      "react/prefer-stateless-function": "off"
+    }
+  },
+  "eslintIgnore": [
+    "build/*"
+  ],
   "devDependencies": {
-    "@babel/core": "^7.1.2",
-    "@webpack-cli/init": "^0.1.2",
-    "babel-core": "^6.26.3",
-    "babel-loader": "^8.0.4",
-    "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "babel-preset-env": "^1.7.0",
-    "css-loader": "^1.0.0",
-    "mini-css-extract-plugin": "^0.4.4",
-    "node-sass": "^4.9.3",
-    "sass-loader": "^7.1.0",
-    "style-loader": "^0.23.1",
-    "webpack": "^4.21.0",
-    "webpack-cli": "^3.1.2",
-    "webpack-dev-server": "^3.1.10"
+    "eslint": "^4.5.0",
```

Comment:
We should add `@webpack-cli/ui-server` to the `devDependencies `

Reply:
Not really I think,  one can use `ui` for testing the gui as a whole.

While `ui-gui` can use `webpack-dev-server` to test while development.

Reply:
Yes, of course. But we also need to have a command to bootstrap the real server. Somewhere the GUI will call some rest API to fetch some data

Reply:
What I was saying was that rest API server can be ran using `webpack-cli ui` this way we wouldn't be cluttering `ui-gui`. Rest depends on what majority feels 😅

Reply:
@ematipico agreed with you, but install will fail since we don't have it published yet. so, for now I'll keep it out and we add in future 👍